### DISCUSSION TITLE
List of Shipped Perl Distributions in the Repository

### DISCUSSION THREAD

**Discussion by rwp0** [StrawberryPerl/Perl-Dist-Strawberry#165] [Q&A] [answered]
I'd like to see a list of Perl distributions (that contain various modules) coming with the Strawberry Perl **without installing it first**.

Is there a file in the source code I can reference or a documentation elsewhere?

We have `DISTRIBUTIONS.txt` in the installation directory, but that's only  after the fact (ie. after installing the software on Windows).

I couldn't find a list on the [website](https://strawberryperl.com), in the [repository](https://github.com/StrawberryPerl/Perl-Dist-Strawberry), and within the [Build.PL](https://github.com/StrawberryPerl/Perl-Dist-Strawberry/blob/master/Build.PL#L36) dependencies.

And doing `dir /s` or `tree` on installation path (which is also after the fact) isn't a good solution either (being ugly and mixed).

---

I've created a gist to copy `DISTRIBUTIONS.txt` for reference, but that's not a global/public solution to the problem:

https://gist.github.com/rwp0/4997d8ebe9cbfad982e1ef47040cd5ed

Could you direct me to a resource listing the shipped distribution if any?

If there's none, maybe it's time to create such a file or listing in the repository?

**Comment by genio** [comment]
https://strawberryperl.com/release-notes/5.38.0.1-64bit.html

**Reply by shawnlaffan** [reply, marked as answer]
@rwp0 - if you want to process the files via a script then the files are also at https://github.com/StrawberryPerl/strawberryperl.com/tree/gh-pages/release-notes
